src/MarginTrading.Brokers/MarginTrading.OrderbookBestPricesBroker/Repositories/OrderbookBestPricesRepository.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Threading.Tasks;
+using AzureStorage;
+using AzureStorage.Tables;
+using Common.Log;
+using Lykke.SettingsReader;
+
+namespace MarginTrading.OrderbookBestPricesBroker.Repositories
+{
+    internal class OrderbookBestPricesRepository : IOrderbookBestPricesRepository
+    {
+        private readonly INoSQLTableStorage<OrderbookBestPricesEntity> _tableStorage;
+
+        public OrderbookBestPricesRepository(IReloadingManager<Settings> settings, ILog log)
+        {
+            _tableStorage = AzureTableStorage<OrderbookBestPricesEntity>.Create(settings.Nested(s => s.Db.HistoryConnString),
```

Comment:
Orderbook best price seems to be more state than history. Maybe it's better to use s.Db.StateConnString.

Reply:
This is not state, but history. The insert method naming was erroneous.